README.md

```diff
@@ -34,6 +39,28 @@ If you don't have the AsyncAPI Generator installed, you can install it like this
 npm install -g @asyncapi/generator
 ```
 
+### or 
+
+### Using AsyncAPI CLI
+
+Install template
+
+```bash
+npm install -g @asyncapi/html-template@0.16.0
```

Comment:
you do not have to install the template of you want to use it, the generator will install it anyway after running `asyncapi generate fromTemplate`.

you do `npm install -g @asyncapi/html-template@0.16.0` only in production, if you first want to make sure the specific version of the template is installed

you could also do `asyncapi generate fromTemplate asyncapi.yaml @asyncapi/html-template@0.16.0` and it should work